### PR TITLE
docs - locale for static and public links

### PR DESCRIPTION
**Pull request by jeff-bruemmer**
How to set locale for static and public links.

Also tacked on a small formatting fix.

### CODE REVIEW

File: docs/permissions/data.md [hunk 1]
```diff
@@ -55,13 +55,13 @@ View data permission settings apply to different levels in your database:
...
 | Can view             | ✅        | ✅      | ✅     |
-| Granular*             | ✅        | ✅      | ❌     |
+| Granular\*           | ✅        | ✅      | ❌     |
 | Sandboxed            | ❌        | ❌      | ✅     |
 | Impersonated         | ✅        | ❌      | ❌     |
 | Blocked              | ✅        | ✅      | ✅     |
 
 
-* The "Granular" setting is not itself a type of permission; it just signals that permissions are set at a level below the current level. For example, you can select "Granular" at a schema level to set permissions per table for tables in that schema.
+\* The "Granular" setting is not itself a type of permission; it just signals that permissions are set at a level below the current level. For example, you can select "Granular" at a schema level to set permissions per table for tables in that schema.
```
Comment (jeff-bruemmer) [inline docs/permissions/data.md:64]: Unrelated fix piggybacking on this PR.

**Review by camsaul — APPROVED** [review]
